cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -1627,13 +1708,13 @@ public JsonObject getTrigger(String id, String supportedPlatform, String balleri
         try {
             Request searchReq = getNewRequest(supportedPlatform, ballerinaVersion)
                     .get()
-                    .url(this.baseUrl + "/" + TRIGGERS + "/" + id)
+                    .url(this.baseUrl + SEPARATOR + TRIGGERS + SEPARATOR + id)
```

Comment:
`TRIGGERS_PATH_SEPARATOR` to hold value of `SEPARATOR + TRIGGERS + SEPARATOR`

Reply:
Fixed with fc2c8abcedf